src/features/main/components/HeroSection/HeroSection.module.scss

```
@@ -0,0 +1,80 @@
+.section {
+  position: relative;
+  overflow: clip;
+  background: url('../../../../assets/images/LoopBg.svg') top right no-repeat;
+  background-size: contain;
+
+  @media (width >=900px) {
+    background-size: auto;
+  }
+}
+
+.section-wrapper {
+  display: flex;
+  align-items: 'center';
+  justify-content: 'space-between';
```

May be we can remove quotes
```suggestion
  align-items: center;
  justify-content: space-between;
```
In this case I don't have errors at my IDE